src/sentry/utils/sentry_apps/request_buffer.py

```diff
@@ -100,18 +113,26 @@ def _get_all_from_buffer(
         else:
             return self.client.lrange(buffer_key, 0, BUFFER_SIZE - 1)
 
-    def _get_requests(self, event: str | None = None, error: bool = False) -> list[dict[str, Any]]:
+    def _get_requests(
+        self, event: str | list[str] | None = None, error: bool = False
```

Comment:
How hard would it be to make event only be a list of strings so we don't have to the check to see if it's one or multiple ? Also idk if there was a discussion or anything about this already

Reply:
mmm I think we can get rid of the `str |` once we get rid of the old endpoint, I'd rather not modify the current endpoint rn